### ISSUE TITLE
Text trivia should not contain '<' or '/>' resp. '/ >'

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#684]
Sometimes, documentation contains remaining of the XML elements, such as in following example:

```c#
/// <summary>
/// Some text/>
/// </summary>
```

We should report them and suggest to remove or replace them.
- `/>` or `/ >` should be removed
- `<` or `>` should be replaced by `&lt;` resp. `&gt;`
